server/core/src/https/views/login.rs

```diff
@@ -981,3 +997,35 @@ fn add_session_cookie(
         })
         .ok_or(OperationError::InvalidSessionState)
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
It's good to have tests, but this test doesn't seem to be exercising anything that could be that fragile? Is the purpose of the test to find if we break the body http or something? I can't see a scenario where this could ever fail ....

Reply:
Mainly just making sure it renders and includes something we expect it to, because actually triggering the error while human-testing is unreliable/hard/bordering on impossible without finding another bug.